src/org/zaproxy/zap/extension/spider/SpiderPanel.java

```diff
@@ -160,6 +163,9 @@
 	/** The found count value label. */
 	private JLabel foundCountValueLabel;
 	
+	private static ZapTableExportButton exportUrlsButton = null;

```

Comment:
I had it that way originally but the enablement and visibility were not properly consistent